mobilizon_reshare/models/publication.py

```diff
@@ -5,9 +5,9 @@
 
 
 class PublicationStatus(IntEnum):
-    WAITING = 1
-    FAILED = 2
-    COMPLETED = 3
+    UNSAVED = 0
```

Comment:
IIUC this is only a placeholder, i believe it could be possible to avoid this by only returning  `EventPublication`s to the user.